tests/bdd/features/volume/resize/test_resize_offline.py

```diff
@@ -221,6 +280,7 @@ def one_of_the_replica_is_not_in_online_state():
     volume = ApiClient.volumes_api().get_volume(VOLUME_UUID)
     # Offline NODE3_NAME to make the replica Offline/Unknown
     Docker.stop_container(NODE3_NAME)
+    time.sleep(1)
```

Comment:
this should use a wait_replica_offline function

Reply:
added a function for that purpose.